src/dispatch.rs

```diff
@@ -169,10 +171,12 @@ impl<'t> Dispatcher<'t> {
 
         #[cfg(target_os = "emscripten")]
         self.dispatch_seq(res);
+
+        self.dispatch_thread_local(res);
```

Comment:
> you can't execute thread local systems while executing 

What do you mean by that? Looking at your PR, it seems to me that you *can* execute thread local systems while the other systems are executed asynchronously, which is what I'd like to avoid.